bin/reth/src/cli/mod.rs

```diff
@@ -232,20 +232,27 @@ impl Logs {
     {
         let mut layers = Vec::new();
 
+        // Function to create a new EnvFilter with base and specific directives
+        let create_env_filter = |additional_directive: &str| -> eyre::Result<EnvFilter> {
+            let mut filter = EnvFilter::builder().from_env_lossy();
+            filter = filter.add_directive(DEFAULT_ENV_FILTER_DIRECTIVE.parse()?);
+            filter = filter.add_directive(additional_directive.parse()?);
+            Ok(filter)
```

Comment:
should work in one statement with builder pattern

```suggestion
            Ok(EnvFilter::builder().from_env_lossy()
                .add_directive(DEFAULT_ENV_FILTER_DIRECTIVE.parse()?)
                .add_directive(additional_directive.parse()?))
```